### ISSUE TITLE
Accessibility issue in Mac OSX

### ISSUE THREAD

**Issue by broseborough** [fnagel/jquery-ui#302]
The select elements do work using the normal behavior of pressing the spacebar, however, this is not the normal 'press' command when using the built in Mac OSX accessibility options.

While a user may eventually figure out that using the spacebar will work, it is not the expected behavior. 

The event handler should be expanded to include 'ctrl+option+space' which is the default 'press' command.

Can I help?

**Comment by fnagel** [comment]
Any help is appriciated. Please feel free to send a pull request!

**Comment by himedlooff** [comment]
@broseborough the spacebar is how I usually navigate default select menus on OSX. Am I missing something?

**Comment by fnagel** [comment]
@broseborough So you say space is default for Mac OS?

Default on most systems is CTRL+DOWN and I not even sure what the "option" key in "ctrl+option+space" is.

**Comment by himedlooff** [comment]
option is the [alt key on Mac](http://en.wikipedia.org/wiki/Option_key)
